src/DataStructures/Tensor/Expressions/ScaleBy.hpp

```diff
@@ -0,0 +1,31 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function scale_by
+
+#pragma once
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+
+/*!
+ * \ingroup Functions
+ * \brief scales a tensor by a scalar
+ *
+ * \returns a tensor of the same type as the input tensor
+ *
+ * \details
+ * Divides the input tensor by the given scalar.
+ */
+template <typename TensorType>
```

Comment:
I changed my mind, writing a ET for `operator*` is more work than we should invest right now. Since these aren't actually expressions but eager, I think we should move them to `Tensor/Eager` or maybe `Tensor/EagerMath` (I have no preference)